README.md

```diff
@@ -50,13 +50,24 @@ Only one instance of the plugin is *possible*.
 - Enabled
   - bool
   - default: false
-  - enable the plugin
+  - Enable the plugin
 - LogLevel
   - string
   - default: `INFO`, expected values are: `INFO`, `DEBUG`
 - CrowdsecMode
   - string
   - default: `live`, expected values are: `none`, `live`, `stream`, `alone`
+- CrowdsecAppsecEnabled
+  - bool
+  - default: false
+  - Enable Crowdsec Appsec Server (WAF) 
+- CrowdsecAppsecScheme
+  - string
+  - default: `http`, expected values are: `http`, `https`
```

Comment:
Do we need another variable for scheme ?  
We use the same TLSconfiguration than the one to contact LAPI.   
If we use a different one we might need more "variables", duplication.  One idea would be to have either HTTP or HTTPS for both and indicate it to the user.

Reply:
As discussed for now we use one shared scheme for LAPI and Appsec